component/pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.ballerinalang</groupId>
         <artifactId>file-connector-parent</artifactId>
-        <version>0.96.1-SNAPSHOT</version>
+        <version>0.97.0-SNAPSHOT</version>
```

Comment:
Isn't it 970?

Reply:
No. This is the connector version. That doesn't need to match with the ballerina runtime version.